### ISSUE TITLE
track creation for plots and track example

### ISSUE THREAD

**Issue by jimmybgammyknee** [maxplanck-ie/HiCExplorer#13]
Links for the Plot TAD section are not available:

```Build Tracks File

We can plot the TADs for a given chromosomal region. For this we need to create a tracks file, which is a file containing instructions to plot. `HERE <>`__ are the instructions on how to build the track file. One of the example is attached here.
```

**Comment by fidelram** [comment]
Sorry for the late reply. The notification went to the wrong folder. I will now fix that.

**Comment by fidelram** [comment]
I had hopefully fixed the problem with commit 28f72e20acf887d10f93a6e0e60ecac7b7e1547a 

Let me know if you need some further help.
